worker/src/test/java/gov/cms/ab2d/worker/processor/coverage/CoverageUpdateAndProcessorTest.java

```diff
@@ -115,7 +115,7 @@ void before() {
 
         CoverageUpdateConfig config = new CoverageUpdateConfig(PAST_MONTHS, STALE_DAYS, STUCK_HOURS);
 
-        processor = new CoverageProcessorImpl(coverageService, bfdClient, taskExecutor, MAX_ATTEMPTS);
+        processor = new CoverageProcessorImpl(coverageService, bfdClient, taskExecutor, MAX_ATTEMPTS, false);
```

Comment:
That's a bit of a code smell, passing in a boolean to a bean.

Better yet, make the PatientFilter a pluggable bean, configured differently for testing vs. production.